Content/Items/Forest/Accessories.AcornSprout.cs

```diff
@@ -25,7 +25,7 @@ public override void Load()
 		public override void Unload()
 		{
             StarlightPlayer.ModifyHitNPCWithProjEvent -= SpawnAcorn;
-        }
+        }   
```

Comment:
Why was this change made? Remind me to run a cleanup tool to remove whitespace like this at some point